src/main/webapp/css/bootstrap.css

```diff
@@ -562,6 +555,15 @@ pre code {
   margin-left: auto;
 }
 
+.feed-container {
```

Comment:
Since you wrote this new feed-container CSS class, I think it would be clearer to keep it in one of the custom, smaller CSS files rather than bundled in here. That way this giant file can just be the plain Bootstrap file as-is.

Reply:
resolved